openid-sharedsignals-framework-1_0.md

```diff
@@ -1431,22 +1439,22 @@ If the stream is:
 
 Enabled
 
-> the Transmitter MUST send a stream updated ({{stream-updated-event}}) event
-  respectively to the Receiver within the Event Stream.
+> the Transmitter MUST send a Stream Updated ({{stream-updated-event}}) event
+  to the Receiver within the Event Stream.
 
 Paused
 
-> the Transmitter SHOULD send a stream updated ({{stream-updated-event}}) after the Event Stream is
+> the Transmitter SHOULD send a Stream Updated ({{stream-updated-event}}) event after the Event Stream is
   re-started. A Receiver MUST assume that events may have been lost during the
-  time when the event stream was paused.
+  time when the Event Stream was paused.
```

Comment:
Should we jus say "Stream" instead of "Event Stream"? I think we have defined "Stream" as a term, so it will be unambiguous to refer to that with the capitalization.

Reply:
```suggestion
  time when the Stream was paused.
```